stdpopsim/models.py

```diff
@@ -276,6 +273,17 @@ def get_samples(self, *args):
                                  f" {pop_index}")
         return samples
 
+    def get_demography_debugger(self):
+        """
+        Returns an msprime DemographyDebugger initialized
```

Comment:
We can link to the docs in msprime using sphinx. Something like

```
Returns an :class:`msprime.DemographyDebugger` instance initialized with the parameters 
for this model. Please see the msprime documentation for details on how to use a 
DemographyDebugger.

:return: A DemographyDebugger instance for this DemographicModel.
:rtype: msprime.DemographyDebugger
```

Reply:
yes- i need to get in this habit! change made